deploy.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+echo "Pull Request: $TRAVIS_PULL_REQUEST"
+echo "Branch: $TRAVIS_BRANCH"
+if [[ $TRAVIS_PULL_REQUEST == "false" ]]; then
+  if [[ $TRAVIS_BRANCH == "master" ]]; then
+    bundle exec nanoc --env=prod deploy public
+  fi
+else
+  rsync -aglpPrtvz --delete output/ "deploy@$TRAVIS_PULL_REQUEST.zeus.werthen.com:/var/www/html/$TRAVIS_PULL_REQUEST/"
```

Comment:
@werthen: Are you gonna preview this @ werthen.com?

Reply:
right now, yeah. I'll be making use of subdomains. Will move this to zeuswpi.org later